kernel/process/process.rs

```diff
@@ -351,6 +356,38 @@ impl Process {
         self.signals.lock().is_pending()
     }
 
+    /// Sets signal mask
+    pub fn rt_sigprocmask(
```

Comment:
How about using a more intuitive name like following?

```suggestion
    pub fn set_signal_mask(
```

Reply:
Done